From b62f020ea723dc33f28e5e865d0ddd13219971c9 Mon Sep 17 00:00:00 2001
From: Cristinel Panfir <cristinel.panfir@nxp.com>
Date: Mon, 29 Mar 2021 15:24:42 +0300
Subject: [PATCH] Makefile: Add v2x test to the installed-binaries

Signed-off-by: Cristinel Panfir <cristinel.panfir@nxp.com>
[regenerate the patch to be applied after meta-imx patches]
Signed-off-by: Fausto Chiatante <fausto.chiatante@nxp.com>

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 25ad333..a397814 100644
--- a/Makefile
+++ b/Makefile
@@ -61,9 +61,9 @@ hsm_doc: include/hsm/hsm_api.h
 	cp doc/latex/refman.pdf doc/hsm_api_document.pdf
 	rm -rf doc/latex/
 
-install: hsm_test she_test she_lib.a seco_nvm_manager.a hsm_lib.a
+install: hsm_test she_test v2x_test she_lib.a seco_nvm_manager.a hsm_lib.a
 	mkdir -p $(DESTDIR)$(BINDIR) $(DESTDIR)$(LIBDIR) $(DESTDIR)$(INCLUDEDIR)
 	cp seco_nvm_manager.a hsm_lib.a she_lib.a $(DESTDIR)$(LIBDIR)
-	cp hsm_test she_test $(DESTDIR)$(BINDIR)
+	cp hsm_test she_test v2x_test $(DESTDIR)$(BINDIR)
 	cp -a include/* $(DESTDIR)$(INCLUDEDIR)
 
-- 
2.25.1

